department.rb

```diff
@@ -0,0 +1,22 @@
+require './employee'
+
+class Department
+  attr_reader :name, :roster
+  def initialize(name)
+    @roster = []
+    @name = name
+  end
+
+  def add_employee(employee)
+    @roster << employee
+  end
+
+  def total_salary
+    total = 0
+    @roster.each do |employee|
+      total += employee.salary
+    end
+    return total
+    # @roster.reduce(0.0) {|sum, employee| sum += employee.salary}
```

Comment:
I'm guessing that you got this reduce from someone else?  You're right, though, that this would be an alternative to what you wrote.